mkdocs/pages/en/devbook/transactions/bonded-aggregate.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Bonded Aggregate
+---
+
+# Creating a Bonded Aggregate Transaction
+
+This tutorial shows how to create an asset swap using <bonded aggregate transactions:>.
+
+In this example, Account A sends 10 <XYM:> to Account B, while Account B simultaneously sends 1 custom <mosaic:> back
+to Account A:
+
+```dot
+digraph {
+    rankdir="LR";
+    fontsize=12;
+    subgraph clusterAggregate {
+        label = "Bonded Aggregate Transaction";
+        tooltip = "Bonded Aggregate Transaction";
+        subgraph clusterT2 {
+            label = "Embedded Transfer 2";
+            tooltip = "Embedded Transfer 2";
+            style = dashed;
+            A1 [label="Account A" tooltip="Account A"];
+            B1 [label="Account B" tooltip="Account B"];
+            A1 -> B1 [label="1 Custom Mosaic" dir=back];
+        }
+        subgraph clusterT1 {
+            label = "Embedded Transfer 1";
+            tooltip = "Embedded Transfer 1";
+            style = dashed;
+            A2 [label="Account A" tooltip="Account A"];
+            B2 [label="Account B" tooltip="Account B"];
+            A2 -> B2 [label="10 XYM"];
+        }
+
+    }
+}
+```
+
+Unlike <complete aggregate transactions:>, bonded aggregates collect <cosignatures:> on-chain.
```

Comment:
Like with the Complete Aggregate tutorial, I think explaining this difference in an info box would be nice.
And also provide a concrete example where a bonded agg works better, e.g., when cosignatures have to be provided by different companies using different tools and programming languages. In this case, the blockchain acts as the common interface.